@types/index.d.ts

```diff
@@ -2,6 +2,6 @@ import config from "config";
 import winston from "winston";
 
 declare global {
-  var config: config;
-  var logger: winston.Logger;
+  let config: config;
```

Comment:
This has to be kept as `var`, to let config and logger be accessible globally. The CI is failing due to this.